sdcflows/interfaces/bspline.py

```diff
@@ -421,6 +426,7 @@ def _run_interface(self, runtime):
             self.inputs.in_data,
             self.inputs.pe_dir,
             self.inputs.ro_time,
+            jacobian=self.inputs.jacobian or False,
```

Comment:
traits to the rescue
```suggestion
            jacobian=self.inputs.jacobian,
```